### PR TITLE
D_E3M9

### PR DESCRIPTION
**Pull request by LolaHThicc**

### CODE REVIEW

**Comment by unithecutie** [comment]
Ah, the file needs to be renamed to and placed under `musics/d_e3m9.mid`.
Also please remember to update CREDITS-MUSIC!

`E3M9: "Noclip to the End" by Lola "BlueWorrior" Harvey`
and under previously featured: `E3M9: "Lipstick" by Jeremy "Vandalorian" Emerson`

**Comment by unithecutie** [comment]
I'm terrible at teaching people things. Sorry Lola! >w<

The file name `d_e3m9.mid` needs to be in lowercase to update it instead of creating a new one.
`./noclip-to-the-end.mid` and `musics/D_E3M9.mid` needs to be deleted, click on the file and delete it here. (don't actually delete the dummy MIDI, that was just an example on how to delete things on your branch)
image

Line 37 of `CREDITS-MUSIC` should be changed to `E3M9: "Noclip to the End" by Lola "BlueWorrior" Harvey`
Line 162 of `CREDITS-MUSIC` should be changed to `E3M9: "Lipstick" by Jeremy "Vandalorian" Emerson`, then create a new line with Shift + Enter.

You can edit text files here.
image

File: D_E3M9.mid
Comment (mc776) [inline D_E3M9.mid:1]: need to delete this file and noclip-to-the-end.mid

File: musics/D_E3M9.mid
Comment (mc776) [inline musics/D_E3M9.mid:1]: unfortunately all this stuff is case-sensitive so what we've got here is a brand new file "D_E3M9.mid" while the old "d_e3m9.mid" is still unreplaced. This also needs to be renamed before the OR should be merged.

**Comment by LolaHThicc** [comment]
thanks for the re-explanation and your patience, i'm still very new to this 😝 hopefully i followed the instructions correctly this time!

**Comment by unithecutie** [comment]
> thanks for the re-explanation and your patience, i'm still very new to this 😝 hopefully i followed the instructions correctly this time!

Yeah, only one thing needs to be fixed: please re-indent line 37 by pressing Tab.

**Review by mc776 — APPROVED** [review]

**Comment by mc776** [comment]
So apparently the branch this PR is based on predates https://github.com/freedoom/freedoom/commit/69166c6728d40e8bbe4c96949cf62f4b914fa84d

I don't know how to fix these conflicts without starting the PR all over again

@selliott512 any idea?

**Comment by selliott512** [comment]
> I don't know how to fix these conflicts without starting the PR all over again
> 
> @selliott512 any idea?

If you squash the commits down to one it looks like only these file will be modified:
```
CREDITS-MUSIC
musics/d_e3m9.mid
```
The conflict is with `musics/d_e3m9.mid`. After the squash you should be able to just replace that file with a version that contains both changes (69166c6 and these changes).

Since it's binary I don't think there is a good automated way of combined the 69166c6 changes with these changes, so one of them will have to be redone. It seems though that you should be able to start with either 69166c6 or these changes as a base, and then redo the other change.

**Comment by mc776** [comment]
Are you able to do it on your end? I don't think anyone else active knows how to squash a thingy without the web interface.

**Comment by selliott512** [comment]
Sure, but I don't know how to come up with a `d_e3m9.mid` that has both the `69166c6` fix, and this fix. If you push a commit here that has that, or attach it as a ZIP, or whatever I can do the git mechanics.

**Comment by mc776** [comment]
Best to do this the old fashioned way then...

**Pull request closed without merging**